src/Telegram/Types/Message.php

```diff
@@ -417,4 +418,16 @@ protected function mapSubObjects(string $key, array $data): TelegramTypes
         // Return always null if none of the objects above matches
         return parent::mapSubObjects($key, $data);
     }
+
+    /**
+     * Decode style entities from Telegram bot messages (bold, italic, etc.) in text with inline entities that duplicate (when possible) the exact style the message had originally when was sended to the bot.
```

Comment:
PHPDocumentor require us to come up with a title that explains in less than 80 chars what a method will do.

In the long description (with 2 enters after the title) you can go as long as you want, while always adhering to the 120 chars line length :)